ompi/mpi/man/man3/Makefile.am

```diff
@@ -39,6 +39,18 @@ MD_FILES = \
         MPI_Close_port.md \
         MPI_Cart_sub.md \
         MPI_Cartdim_get.md \
+        MPI_Comm_accept.3in \
```

Comment:
These should all me `.md`, not `.3in`.  This is what (rightfully) caused Mellanox CI to fail.